XRTK-Core/Packages/com.xrtk.core/Providers/Controllers/UnityInput/MouseController.cs

```diff
@@ -106,6 +105,8 @@ public void Update()
 
                 if (Interactions[i].InputType == DeviceInputType.Scroll)
                 {
+                    Interactions[i].InvertXAxis = true;
+                    Interactions[i].InvertYAxis = true;
```

Comment:
This ought to be configurable

Reply:
I suppose, but as a single switch (invert on Scroll) or are you thinking of a global invert option?

Reply:
Probably part of the input mapping as an option next to the action dropdown. (same for dual axis actions as well)